README.md

```diff
@@ -1 +1,3 @@
-# coffee-roaster-market
\ No newline at end of file
+[![CircleCI](https://circleci.com/gh/kuchichan/coffee-roaster-market.svg?style=svg)](https://circleci.com/gh/kuchichan/coffee-roaster-market)
```

Comment:
Probably you will need to sing up for circle Ci and change here a little bit (kuchichan -> Aleksander Pawlak) and also change settings to protect branch from merging without passing the pipelines.

Reply:
Here you can check how it looks (click on green tick, you can also look at badge) https://github.com/kuchichan/coffee-roaster-market/tree/feature/circleci